packages/radio-group/stories/radio-group.stories.mdx

```diff
@@ -176,8 +176,8 @@ const validate = () => {
 
 ```html
 <lion-radio-group
-  id="dinos"
-  name="dinos"
+  id="dinosGroup"
```

Comment:
Reason for this change? We kinda changed the way we handle `name` attributes and in most places I think we changes `{x}Group` to just `{x}s`

Reply:
I saw that we were using document.querySelector('#dinosGroup'); in line 172, so i changed the id and since i had to change the name to something in order to fix this issue I kept the name same as id

Reply:
Ah fair enough, fine with me then, as long as it works